synchronization/src/main/AndroidManifest.xml

```diff
@@ -41,7 +41,8 @@
                     android:resource="@xml/sync_adapter" />
         </service>
         <service android:name="de.cyface.synchronization.AuthenticatorService"
-            android:exported="false">
+            android:exported="false"
+            android:process=":sync"> <!-- FIXME: experimental, probably just forgot to do this -->
```

Comment:
Does it make sense to run the AuthenticatorService in the same process as :sync  or should this be in the default process as right now (:ui)?